include/swift/Parse/ParsedRawSyntaxNode.h

```diff
@@ -196,30 +194,18 @@ class ParsedRawSyntaxNode {
     case DataKind::Null:
       break;
     }
+    copy.Range = Range;
     copy.SynKind = SynKind;
     copy.TokKind = TokKind;
     copy.DK = DK;
     copy.IsMissing = IsMissing;
     return copy;
   }
 
-  CharSourceRange getDeferredRange() const {
-    switch (DK) {
-    case DataKind::DeferredLayout:
-      return getDeferredLayoutRange();
-    case DataKind::DeferredToken:
-      return getDeferredTokenRangeWithTrivia();
-    default:
-      llvm_unreachable("node not deferred");
-    }
-  }
+  CharSourceRange getRange() const { return Range; }
```

Comment:
While you are here, please add a comment that this includes trivia ranges.

Reply:
I’ll do.